.circleci/config.yml

```diff
@@ -103,6 +103,23 @@ commands:
       - run:
           name: Running network tests
           command: npm run test:docker
+  extension-server-start:
+    steps:
+      - run:
+          name: "Extension: server start"
+          background: true
+          command: |
+            echo 'export APPLICATION_SID_EXTENSION="$APPLICATION_SID_EXTENSION"' >> "$BASH_ENV"
+            echo 'export ACCOUNT_SID="$ACCOUNT_SID"' >> "$BASH_ENV"
+            echo 'export API_KEY_SID="$API_KEY_SID"' >> "$BASH_ENV"
+            echo 'export API_KEY_SECRET="$API_KEY_SECRET"' >> "$BASH_ENV"
+            source "$BASH_ENV"
+            npm run start:extension:server
```

Comment:
Can we put this in a script file and call the script here instead? See how we're doing relay server. Also, once it's in a script, do we still need these environment variables to be exported?

Reply:
Forgot to clean this up, export not necessary